scripts/transforms/README.md

```diff
@@ -1,16 +1,18 @@
 ## Codemods
 
-These codemods should be ran with [`jscodeshift`](https://github.com/facebook/jscodeshift).
+These codemods (code transformers) can be ran with `npx`. If you installed the `instantsearch.js` package already with `yarn`, you can run them like this: `yarn instantsearch-scripts`).
 
-### addWidget / removeWidget
+### `addWidget-to-addWidgetss`
 
-These should be replaced to `addWidgets` and `removeWidgets`.
+This will replace calls to `addWidget(widget)` to `addWidgets([widget])`.
 
 ```
-yarn jscodeshift --transform scripts/transforms/addWidget-addWidgets.js --extensions='ts,js,tsx' examples stories .storybook
+npx instantsearch.js addWidget-to-addWidgets <path>
 ```
 
-Make sure to run Prettier after!
+### Notes
+
+If you are using Prettier or eslint, make sure to run its autofixing after this transformation, since it's possible that code should be formatted differently after it has been transformed. For example in our repository using prettier that would be:
```

Comment:
```suggestion
If you are using Prettier or ESLint, make sure to run its autofixing after this transformation, since code can be formatted differently after it has been transformed. For example, in our repository, the Prettier command would be:
```